docs/content/en/docs/getting-started.md

```diff
@@ -0,0 +1,466 @@
+  annotations:
+    keptn.sh/lifecycle-toolkit: enabled
+```
+
+Create the namespace:
+
+```shell
+kubectl apply -f namespace.yaml
+```
+
+## Deploy Demo Application
+
+It is time to deploy the demo application.
+
+Save this manifest as `app.yaml`:
+
+```shell
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deployment
+  namespace: keptndemo
+  labels:
+    app.kubernetes.io/name: nginx
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: nginx
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/part-of: keptndemoapp
+        app.kubernetes.io/name: nginx
+        app.kubernetes.io/version: 0.0.1
+    spec:
+      containers:
+      - name: nginx
+        image: nginx:1.14.2
+        ports:
+        - containerPort: 80
+---
+apiVersion: apps/v1
+kind: Service
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx
+  namespace: keptndemo
+spec:
+  selector:
+    app.kubernetes.io/name: nginx
+  ports:
+    - protocol: TCP
+      port: 8080
+      targetPort: 80
+```
+
+Now apply:
+
+```shell
+kubectl apply -f app.yaml
+```
+
+Keptn looks for these 3 labels:
+
+- `app.kubernetes.io/part-of`
+- `app.kubernetes.io/name`
+- `app.kubernetes.io/version`
+
+These are [Kubernetes recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels)
+but if you want to use different labels, you can swap for:
+
+- `keptn.sh/app` instead of `app.kubernetes.io/part-of`
+- `keptn.sh/workload` instead of `app.kubernetes.io/name`
+- `keptn.sh/version` instead of `app.kubernetes.io/version`
+
+## Explore Keptn
+
+Keptn is now aware of your deployments and is generating DORA statistics about them.
+
+Keptn has created a CRD to track your application.
+The name of which is based on the `part-of` label.
+
+It may take up to 30 seconds to create the `KeptnApp` so run the following command until you see the `keptnappdemo` CRD.
+
+```shell
+kubectl -n keptndemo get keptnapp
+```
+
+Expected output:
+
+```shell
+NAME           AGE
+keptndemoapp   2s
+```
+
+Keptn also creates a new application version every time you increment the `version` label..
+
+The `PHASE` will change as the deployment progresses.
+A successful deployment is shown as `PHASE=Completed`
+
+```shell
+kubectl -n keptndemo get keptnappversion
+```
+
+Expected output:
+
+```shell
+NAME                      APPNAME        VERSION   PHASE
+keptndemoapp-0.0.1-***    keptndemoapp   0.0.1     Completed
+```
+
+Keptn can run tasks and SLO evaluations before and after deployment.
+You haven't configured this yet, but you can see the full lifecycle for a `keptnappversion` by running:
+
+```shell
+kubectl -n keptndemo get keptnappversion -o wide
+```
+
+Keptn applications are a collection of workloads.
+By default, Keptn will build `KeptnApp` CRDs based on the labels you provide.
+
+In the example above, the `KeptnApp` called `keptndemoapp` contains one workload (based on the `name` label):
```

Comment:
How hard would it be to include two workloads in this demo?  That just seems like a good way to emphasize that one KeptnApp can include multiple workloads.

Reply:
We could easily switch out the demo to use the OpenTelemetry demo (which seems to be moving towards the industry standard demo app) BUT it requires 4GB just for the demo app.

We could also use [podtato head](https://github.com/podtato-head/podtato-head) as that's the standard from the TAG App Delivery folk.